packages/pglite/src/utils.ts

```diff
@@ -72,28 +75,47 @@ export const uuid = (): string => {
 }
 
 export async function formatQuery(
-  pg: PGliteInterface | Transaction,
+  pg: PGliteInterface,
```

Comment:
might be worth documenting this function, passing both pg and a transaction feels odd - I'm assuming this is because you can still execute the raw protocol while holding the transaction lock and you need it to describe the parameters?